### DISCUSSION TITLE
Create sub modules into created external modules

### DISCUSSION THREAD

**Discussion by dchiragj** [oqtane/oqtane.framework#1184] [Q&A]
Hello Everyone,

I need to create one external module with submodules, So can anyone guide me how do we can create submodules into external modules.

**Comment by hishamco** [comment]
Could you please elllaborate what is the need to have sub modules inside a module? Is there a certain scenario so other contributors can understand and give you a proper answer

**Comment by sbwalker** [comment]
@dchiragj this thread might be helpful https://github.com/oqtane/oqtane.framework/issues/1100
